src/autowiring/test/PostConstructTest.cpp

```diff
@@ -385,3 +385,22 @@ TEST_F(PostConstructTest, PostConstructCanSafelyThrow) {
   Autowired<PostConstructThrowsException> pcte;
   ASSERT_FALSE(pcte.IsAutowired()) << "A context member which threw an exception post-construction was incorrectly introduced into a context";
 }
+
+namespace {
+  class EmptyType : public CoreObject {};
+}
+
+TEST_F(PostConstructTest, StrictNotificationArrangement) {
+  size_t call[3] = {0, 0, 0};
```

Comment:
Recommend you initialize these to something other than `0` so we can tell if the lambdas weren't even called.